source/deployment/advanced-permissions.rst

```diff
@@ -0,0 +1,191 @@
+
+- The permissions granted in a Team Override Scheme apply only in the teams which are assigned to the scheme. 
+- The System Scheme does not apply to teams that are added to a Team Override Scheme.
+- Teams can only belong to one Team Override Scheme.
+
+**Scheme List Interface** 
+
+List of all Team Override Schemes in the system. Sorted alphabetically by scheme name. Also displayed is the scheme description and tags with all asociated teams. Available in System Console > Advanced Permissions > Permissions Schemes:
+
+.. image:: ../images/team-scheme-list.png
+
+**Team Override Scheme Interface** 
+
+Interface for naming, assigning teams and editing permissions in a Team Override Scheme, with panels for All members, Channel Admins and Team Admins. Available in System Console > Advanced Permissions > Permissions Schemes > Team Override Scheme:
+
+.. image:: ../images/team-scheme.png
+
+Channel Override Permissions (E20)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+*Available in a future release of Enterprise Edition E20*
+
+Allows Admins to restrict permissions within specific channels. Permissions under consideration for this phase include:
+
+- **Read-only Channels:** The ability for Admins to turn off posting in specified channels.
+- **Restrict Channel Mentions:** Turn off the ability for users to post channel wide mentions (@-all/channel/here) in specified channels.
+- **Channel member management:** Restricting adding and removing channel members to Admins only in specified channels.
+
+Supplementary Roles (E20)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+*Available in a future release of Enterprise Edition E20*
+
+Allows Admins to grant additional permissions to specific users or to a group of users based on AD/LDAP group membership. Permissions can be granted within the scope of channels, teams or system level.
+
+Recipes
+--------
+This section provides some examples of common permissions use cases and how to accomplish them using the Advanced Permissions System Console interface.
+
+Team Management
+~~~~~~~~~~~~~~~~
+
+**Restict adding members to only Admins, in specific teams**
+Example: In Team A, only allow Team and System Admins to add new team members. As the default for all other teams, allow all users to add and invite new members.
+
+1. In **System Console** > **Advanced Permissions**, edit the **System Scheme**
+2. In the **All Members** panel, check the box for **Add Team Members**. This sets the system default for all teams.
+3. In **System Console** > **Advanced Permissions**, create a new **Team Override Scheme**
+  i. Name and describe the scheme. For example, ``Authorized Personel Only`` with description ``Restrict adding team members to Team and System Admins.``
+  ii. Add Team B to the **Select teams to override permissions** list.
+  iii. In the **All Members** panel, uncheck the box for **Add Team Members**.
+  iv. In the **Team Admin** and **Channel Admin** panels, check the box for **Add Team Members**. 
+
```

Comment:
P2/nice-to-have: screenshots are nice too. E.g. showing a screenshot after step iv.